apps/lfx-one/src/app/modules/dashboards/executive-director/components/paid-social-reach-drawer/paid-social-reach-drawer.component.html

```
@@ -0,0 +1,163 @@
+        aria-label="Close panel"
+        data-testid="paid-social-reach-drawer-close">
+        <i class="fa-light fa-xmark text-xl"></i>
+      </button>
+    </div>
+  </ng-template>
+
+  <!-- Content -->
+  <div class="flex flex-col gap-6 pb-2" data-testid="paid-social-reach-drawer-content">
+    <!-- Summary Stats -->
+    <div class="flex gap-4" data-testid="paid-social-reach-drawer-stats">
+      @if (data().roas > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)